src/helix-ui/styles/components/pills.less

```diff
@@ -5,9 +5,13 @@ hx-pill {
 
 hx-status {
   background-color: transparent;
-  border: 1px solid @gray-750;
+  border-color: @gray-750;
```

Comment:
Revert all changes to this file. CSS Custom Properties are moot in the Light DOM.